integrate locks into fiosdk_kotlin

edrotthoff opened the pull request:
integrate locks into fiosdk_kotlin

Commit:
integrate locks into fiosdk_kotlin

integrate locks into fiosdk_kotlin

ericbutz commented on fiosdk/src/main/java/fiofoundation/io/fiosdk/models/fionetworkprovider/LockPeriod.kt:
```diff
@@ -0,0 +1,10 @@
+package fiofoundation.io.fiosdk.models.fionetworkprovider
```
Duplicate file

ericbutz commented on app/src/androidTest/java/fiofoundation/io/fiokotlinsdktestapp/DevSdkTests.kt:
```diff
@@ -11,6 +11,7 @@ import fiofoundation.io.fiosdk.errors.FIOError
 import fiofoundation.io.fiosdk.errors.session.TransactionBroadCastError
```
Eric to review

ericbutz replied:
Reviewed the tests. We do not want to go overboard with tests, but it would be good to have one positive and one negative test. I would suggest:
1. Modify this test to include multiple lock periods.
2. Create a test that catches trying to add a lock to an existing account.

edrotthoff replied:
I will extend these tests to do those...

Commit:
clean up extra copy of lockperiod.

clean up extra copy of lockperiod

Commit:
add modified tests

add test for mulitple locking periods, and also test failure to send to pre-existing account

ericbutz reviewed (approved):
Changes look good.
